packages/common/object/src/map-object.ts

```diff
@@ -0,0 +1,16 @@
+import { PlainObjectOf } from '@angular-ru/common/typings';
+
+export function mapObject<Type, ReturnType>(
+    object: PlainObjectOf<Type>,
+    mapper: (obj: Type, key: string) => ReturnType
+): PlainObjectOf<ReturnType> {
+    const result: PlainObjectOf<ReturnType> = {};
+
+    for (const key in object) {
+        if (object.hasOwnProperty(key)) {
```

Comment:
when can you get "false" here?

Reply:
@StepaZol 

![image](https://user-images.githubusercontent.com/12021443/124084716-f1e10d00-da57-11eb-8f45-38ee40c3cdd1.png)

Reply:
Looping over objects with a for in loop will include properties that are inherited through the prototype chain. This behavior can lead to unexpected items in your for loop.

![image](https://user-images.githubusercontent.com/12021443/124084851-16d58000-da58-11eb-9c23-be79ba394e53.png)

Reply:
https://eslint.org/docs/rules/guard-for-in

Reply:
oh yep. I forgot about it

Reply:
when we got a key from prototype chain. 

ESLint: The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.(guard-for-in)